rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictParamRector/Fixture/skip_call_by_ref.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictParamRector\Fixture;
+
+class SkipCallbByRef {
+    public function doFoo(SkipCallbByRef $param) {
```

Comment:
is this a case worth covering? do other `ReturnTypeFromStrict` rules take into account that a variables type might change because it is passed by ref into a function/method?

Reply:
ok, that one was easier then expected ;)